about.html

```diff
@@ -0,0 +1,147 @@
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="./about.css">
+    <title>About Page</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <div class="logo">
+          <a href="index.html"><img class="logo-img" src="images/logo.png" alt="logo icon"></a>
+        </div>
+        <ul class="navlist">
+          <li><img src="images/close-btn.png" id="close" class="closeIcon" alt="closing icon"></li>
+          <li class="nav-item">
+            <a class="nav-link" href="index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="about.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#contact">Contact</a>
+          </li>
+        </ul>
+        <img src="images/hamburger.png" class="hamburger" id="hamburger" alt="hamburger menu">
+      </nav>
+    </header>
+    <section class="about-page">
+      <div class="container">
+        <h1 class="big-text">ABOUT&nbsp;US</h1>
+        <h4 class="small-text">GAMESCOM IS A PLATFORM FOR AMERICAN DIGITAL CREATIVES ,
+           CELEBRATING THEIR WORKS AND CONNECTING THEM TO OPPORTUNITIES FOR 
+           LEARNING, .
+          COLLABORATION AND FUNDING..
+        </h4>
+        <p class="gamescom-p">
+          GAMESCOM is a platform that brings together enthusiasts and digital creatives across the games, animation, comics, 
+          and movie industry in America via insightful events such as conventions, game jams and so much more.
+          Our aim is to invite investments into the space and foster collaboration among digital creatives across America as well as celebrating the rich and amazing level of creativity possessed by Americans.
+        </p>
+        <button type="button" class="recap">2020 RECAP</button>
+        <div class="about-email">
+          <p>
+            Please contact us on this Email for any further questions or 
+            suggestions about GAMESCOM event 2021!
+          </p>
+          <p><a href="">gamescom@gamemail.com</a></p>
+        </div>
+      </div>
+    </section>
+    <section class="past-gamescom">
+      <h2>View past GAMESCOM Events</h2>
+      <hr>
+      <p>Take a look at the last two GAMESCOM Events which took place in
+         America and Spain
+      </p>
+      <div class="last-events">
+        <div class="events-container">
+          <div class="events">
+            <h3>2019</h3>
+            <p>GAMESCOM Event 2019 in America</p>
+          </div>
+        </div>
+        <div class="events-container">
+          <div class="events">
+            <h3>2019</h3>
+            <p>GAMESCOM Event 2020 in Spain</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Please give the images more width to look like the original design. 

![Capture](https://user-images.githubusercontent.com/64173189/142464065-df65c439-86f3-46e5-84e2-540e78fd3029.JPG)


Original design 

![Capture](https://user-images.githubusercontent.com/64173189/142464130-0540b377-47c3-407d-a796-19284a3ceb8d.JPG)